Inventory.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace InventoryManagmentMongoDB
+{
+    public class Inventory
```

Comment:
this class name is misleading as it resembles a model name. It would be more appropriate to rename it toInventoryRepository -as I suggested earlier-. This way, you can include all methods that interact with the db context directly within this class.